kgraphql-ktor-stitched/src/main/kotlin/com/apurebase/kgraphql/stitched/schema/dsl/StitchedSchemaConfigurationDSL.kt

```diff
@@ -0,0 +1,37 @@
+package com.apurebase.kgraphql.stitched.schema.dsl
+
+import com.apurebase.kgraphql.ExperimentalAPI
+import com.apurebase.kgraphql.schema.dsl.SchemaConfigurationDSL
+import com.apurebase.kgraphql.stitched.schema.configuration.StitchedSchemaConfiguration
+import com.apurebase.kgraphql.stitched.schema.execution.RemoteArgumentTransformer
+import com.apurebase.kgraphql.stitched.schema.execution.RemoteRequestExecutor
+import com.fasterxml.jackson.databind.DeserializationFeature
+
+@ExperimentalAPI
+open class StitchedSchemaConfigurationDSL : SchemaConfigurationDSL() {
+    // Remote executor has to be set for remote schemas
+    var remoteExecutor: RemoteRequestExecutor? = null
+
+    // Local url has to be set when stitching local queries (only)
+    var localUrl: String? = null
+
+    override fun build(): StitchedSchemaConfiguration {
+        objectMapper.configure(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY, acceptSingleValueAsArray)
+        return StitchedSchemaConfiguration(
+            useCachingDocumentParser = useCachingDocumentParser,
+            documentParserCacheMaximumSize = documentParserCacheMaximumSize,
+            objectMapper = objectMapper,
+            useDefaultPrettyPrinter = useDefaultPrettyPrinter,
+            coroutineDispatcher = coroutineDispatcher,
+            wrapErrors = wrapErrors,
+            executor = executor,
+            timeout = timeout,
+            introspection = introspection,
+            plugins = plugins,
+            genericTypeResolver = genericTypeResolver,
+            argumentTransformer = RemoteArgumentTransformer(),
+            remoteExecutor = requireNotNull(remoteExecutor) { "Remote executor not defined" },
```

Comment:
What if (for whatever reason) only local schemata are stitched?
I'm thinking about whether it could be a good idea to have the executor as part of the stitched schema definition. I'm roughly thinking about something like an interface `StitchedSchema` and two concrete implementations `RemoteStitchedSchema` and `LocalStitchedSchema`. The last one could possibly be test-scope only, because I don't really see a production case for having that.

This way we'd have two concrete implementations instead of two nullable values in this ConfigurationDSL.

Maybe I get the concept wrong and we'd need an abstract `StitchedSchemaConfigurationDSL` and two concrete implementations `RemoteStitchedSchemaConfigurationDSL` and `LocalStitchedSchemaConfigurationDSL`, I'm not entirely sure.